libs/MailDB/include/MailDB/PgMailDB.h

```diff
@@ -17,32 +17,20 @@ class PgMailDB : public IMailDB
 
     virtual ~PgMailDB();
 
-    // TODO: Viacheslav
     virtual void Connect(const std::string &connection_string) override;
```

Comment:
Redundant 'virtual' specifier on an overriding function in each function of this translation unit